### PR TITLE
feat: Redirect to application page on success

### PR DESCRIPTION
**Pull request by inderps**
Backend PR
https://github.com/GenomicDataInfrastructure/gdi-userportal-access-management-service/pull/48

### CODE REVIEW

File: src/app/basket/page.tsx [hunk 1]
```diff
@@ -32,12 +32,9 @@ export default function Page() {
...
   const requestNow = async () => {
     try {
-      await createApplication(basket.map((dataset) => dataset.id));
-      setAlert({
-        message: "Application created successfully",
-        type: "success",
-      });
+      const response = await createApplication(basket.map((dataset) => dataset.id));
       emptyBasket();
+      window.location.href = `/applications/${response.applicationId}`;
```
Comment (admy7) [inline src/app/basket/page.tsx:37]: We can use the redirect function from Next instead of manipulating directly the window object

Comment (inderps) [reply src/app/basket/page.tsx:37]: @admy7 
I tried redirect approach first but it was not working. Maybe we can take a look at it together tomorrow

Comment (brunopacheco1) [reply src/app/basket/page.tsx:37]: @inderps @admy7 as this is a tiny thing, I hope you don't mind that approve and merge this PR. Can you please align and open another PR later?

**Comment by brunopacheco1** [comment]
@inderps can you please check the tests?

File: src/app/basket/page.tsx (hunk 1 above)
Comment (admy7) [reply src/app/basket/page.tsx:37]: Np

**Review by admy7 — APPROVED** [review]

**Pull request merged**